src/cases/containers/query-management-container/query-management-container.component.spec.ts

```diff
@@ -94,20 +104,31 @@ describe('QueryManagementContainerComponent', () => {
     expect(locationMock.back).toHaveBeenCalled();
   });
 
+  it('should navigate to qualifying questions selection page', () => {
+    component.queryCreateContext = QueryItemType.NEW_QUERY_QUALIFYING_QUESTION_DETAIL;
+    component.previous();
+    expect(component.queryCreateContext).toEqual(QueryItemType.NEW_QUERY_QUALIFYING_QUESTION_OPTIONS);
+  });
+
   describe('when it does not have a query id', () => {
     it('should not set the query item', () => {
       expect(component.queryItem).toBeUndefined();
     });
 
-    it('should have the ccd-query-write-raise-query component', () => {
+    it('should set the queryCreateContext to be query item type of new query qualifying question options', () => {
+      expect(component.queryCreateContext).toEqual(QueryItemType.NEW_QUERY_QUALIFYING_QUESTION_OPTIONS);
+    });
+
+    it('should have the ccd-qualifying-question-options component', () => {
+      console.log('SUMMARY', component.showSummary);
```

Comment:
@johnbenjamin-hmcts - can remove this `console.log` statement

Reply:
Thank you @Daniel-Lam, I have updated the code now.